zerver/views/users.py

```diff
@@ -110,6 +123,7 @@ def patch_bot_backend(request, user_profile, email,
                       default_sending_stream=REQ(default=None),
                       default_events_register_stream=REQ(default=None),
                       default_all_public_streams=REQ(default=None, validator=check_bool)):
+    # type: (HttpRequest, UserProfile, text_type, text_type, Stream, Stream, bool) -> HttpResponse
```

Comment:
I think `default_sending_stream` and `defaut_events_register_stream` are actually the names of the streams, and thus strings.